src/main/java/edu/ucsb/cs156/courses/repositories/PersonalScheduleRepository.java

```diff
@@ -12,5 +12,4 @@
 public interface PersonalScheduleRepository extends CrudRepository<PersonalSchedule, Long> {
   Optional<PersonalSchedule> findByIdAndUser(long id, User user);
   Iterable<PersonalSchedule> findAllByUserId(Long user_id);
-  Optional<PersonalSchedule> findByUserAndNameAndQuarter(User user, String name, String quarter);
```

Comment:
Why are you removing this?     Removing code needs a justification.  Are you absolutely sure that it's not being used anywhere else in the code base?